lib/js/components/Buttons/ToggleButton/ToggleButton.vue

```diff
@@ -0,0 +1,322 @@
+<template>
+	<ds-ripple
+		class="toggleButtonWrapper"
+		:disable="!isInteractive"
+		:color="rippleColor"
+		@click="$emit('click')"
```

Comment:
I believe it doesn't do anything.
```suggestion
```

Reply:
with this line you don't have to attach event handlers as `native` when using this component

Reply:
But you already have https://github.com/bethinkpl/design-system/blob/5350d819d6ea7b00a3591265f9daef9213de0e94/lib/js/components/Buttons/ToggleButton/ToggleButton.vue#L29

Reply:
fotgot about it :facepalm: